docs/shipping/Code/nestjs.md

```diff
@@ -329,6 +329,10 @@ logzio-k8s-telemetry logzio-helm/logzio-k8s-telemetry
 
 #### Define the logzio-k8s-telemetry service dns
 
+:::note
+If you're using the unified logzio-monitoring chart, update the service to point to the specified host: `http://logzio-monitoring-otel-collector.monitoring.svc.cluster.local:4317/`
+:::
+
 In most cases, the service dns will be `logzio-k8s-telemetry.default.svc.cluster.local`, where `default` is the namespace where you deployed the helm chart and `svc.cluster.name` is your cluster domain name.
```

Comment:
The service for telemetry is logzio-k8s-telemetry-otel-collector.default.svc.cluster.local

We should clarify the service for monitoring chart is also depended on the namespace the chart is installed on and its name, so it can be a placeholder both both cases `http://<<CHART-NAME>>-otel-collector.<<NAMESPACE>>.svc.cluster.local:4317`